doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1129,6 +1140,73 @@ The subscription is represented by a JSON object as described at the beginning o
 
             Location: /v2/subscriptions/abcde98765
 
+### Get subscription [GET /v2/subscription/{subId}]
+
+The response is the subscription represented by a JSON object as described at the beginning of this section.
+
+Response:
+
+* Successful operation uses 200 OK
+* Errors use a non-200 and (optionally) an error payload. See subsection on Error Responses for more detail.
+
++ Parameters
+    + subId: abcdef (required, string) - subscription Id.
+
++ Response 200 (application/json)
+
+        {
+            "id": "abcdef",
+            "subject": {
+                "type": "Room",
+                "condition": {
+                    "attributes:" [ "temperature " ],
+                    "expression:" {
+                       "q": "tempature>40"
+                    }
+                 }
+            },
+            "notification": {
+                "callback": "http://localhost:1234",
+                "attributes": ["temperature", "humidity"],
+                "throttling": "PT5S"
+            },
+            "duration": "PT1M"
+        }
+
+### Update subscription [PATCH /v2/subscription/{subId}]
+
+Only the fields included in the request are updated in the subscription.
+
+Response:
+
+* Successful operation uses 204 No Content
```

Comment:
Not sure if 200 OK without payload is legal... Anyway, using 204 No Content (that seems have been invented for that purpose given its name :) make that more explicit.

NTC